### PR TITLE
Fix: minor relative path issues for the dockerfile and build args

### PR DESCRIPTION
**Pull request by derrandz**
Changes:

 - Fix dockerfile path to be relative in relation to the provided context path
 - Fix POCKET_E2E_STACK_RELATIVE_PATH build arg
 - Add convenience function to export .env for the user on bin/pokt-net-dev-tm.sh
 - Add a new pokt-net env to allow for single node running to scenarios when the user is interested in pocket-cli or anything else related to using a single node
 
 
 Todo:
 
  - [ ] Add a comment to clarify that the env: rpc for the stack: pokt-net won't be needed if you have a pocket-cli installed on your machine

### CODE REVIEW

File: bin/pokt-net-dev-tm.sh [hunk 1]
```diff
@@ -1,5 +1,23 @@
...
+
+  source .env
+  export CWD=$CWD
+  export POCKET_CORE_REPOS_PATH=$POCKET_CORE_REPOS_PATH
+  export POCKET_CORE_REPO_PATH=$POCKET_CORE_REPO_PATH
+  export POCKET_NETWORK_TENDERMINT_PATH=$POCKET_NETWORK_TENDERMINT_PATH
+  export POCKET_E2E_STACK_PATH=$POCKET_E2E_STACK_PATH
+  export DEBUG=$DEBUG
+}
+
```
Comment (derrandz) [inline bin/pokt-net-dev-tm.sh:20]: For convenience, makes life way easier

File: bin/pokt-net-dev-tm.sh [hunk 2]
```diff
@@ -42,9 +61,17 @@ function check_docker() {
...
+source_and_export_env
 check_required_env_vars
 check_docker
 update_chains_json
 
+
+action=$1
 # Call the pokt-net stack.
-./bin/pkt-stack pokt-net dev-tm up
+if [[ -z $action ]];
```
Comment (derrandz) [inline bin/pokt-net-dev-tm.sh:72]: Was useful during the debugging of paths, I was using a lot of `config` but it's totally optional

**Comment by Olshansk** [comment]
Seems like this PR had the RPC stack included in it as well?

File: bin/pokt-net-dev-tm.sh (hunk 1 above)
Comment (Olshansk) [reply bin/pokt-net-dev-tm.sh:20]: I see these scripts as an evolving work in progress, so w/e makes life easier is great. We'll refactor eventually.

Also, I use zshell with the dotenv plugin. Makes sourcing these really easy.

**Pull request merged**